packages/ui-library/package.json

```diff
@@ -68,10 +77,9 @@
     "classnames": "^2.3.2",
     "lodash": "^4.17.21",
     "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
shouldn't these now be a peer dependency?

Reply:
I didn't due to the Storybook dev dep. But you're right, library first this is better so I changed it.